api-extractor/gulpfile.js

```diff
@@ -2,4 +2,6 @@
 
 const build = require('@microsoft/node-library-build');
 
+build.typescript.taskConfig.typescript = require('typescript');
```

Comment:
This changes the version that is compiling the code to JS, but the version of TS that the unit tests are using seems to be wrong...